packages/image_picker/image_picker_platform_interface/lib/src/types/picked_file/io.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import './base.dart';
+
+/// A PickedFile backed by a dart:io File.
+class PickedFile extends PickedFileBase {
```

Comment:
Why do we need 2 different `PickedFile`s?

Reply:
Yes, we need them because only the appropriate one gets conditionally exported by the picked_file.dart file.